test/index-test.js

```diff
@@ -110,6 +110,14 @@ tests = [{
     m: 'i'
   , o: [{i: 1, o: 2}, {i: 2, o: 2}]
   , e: [{i: 1}, {i: 2}]
+}, {
+    m: 'foo(bar)'
+  , o: { foo: { biz: 'baz' } }
+  , e: { foo: {} },
+}, {
+    m: 'foo(bar)'
+  , o: { foo: 1234 }
+  , e: { foo: {} }
```

Comment:
Agree, this should be `null`. Actually this was fixed in line https://github.com/nemtsov/json-mask/pull/9/files#diff-77a92250315fdf44b4ecf5b3fcf5fd3cR64 but the change in line https://github.com/nemtsov/json-mask/pull/9/files#diff-77a92250315fdf44b4ecf5b3fcf5fd3cR38 converts the `null` to `{}`. I was attempting to fix a separate issue when `1234` would not be masked.

Should I separate this into another PR?